inc/class-wpseo-replace-vars.php

```diff
@@ -1091,21 +1097,19 @@ public static function get_advanced_help_texts() {
 	/**
 	 * Set the help text for a user/plugin/theme defined extra variable.
 	 *
-	 * @param  string $type      Type of variable: 'basic' or 'advanced'.
-	 * @param  string $replace   Variable to replace, i.e. '%%var%%'.
-	 * @param  string $help_text The actual help text string.
+	 * @param  string                       $type                   Type of variable: 'basic' or 'advanced'.
+	 * @param  WPSEO_Replacement_Variable   $replacement_variable   The replacement variable to register.
 	 */
-	private static function register_help_text( $type, $replace, $help_text = '' ) {
-		if ( is_string( $replace ) && $replace !== '' ) {
-			$replace = self::remove_var_delimiter( $replace );
+	private static function register_help_text( $type, WPSEO_Replacement_Variable $replacement_variable ) {
+		$replacement_variable->set_variable( self::remove_var_delimiter( $replacement_variable->get_variable() ) );
```

Comment:
Perhaps you can move this: `self::remove_var_delimiter(...` to the declaration of this object in line `118`

Reply:
And if you moved that, you can remove the `set_variable` method as well